src/interpreter/kernel/exec.js

```diff
@@ -703,6 +705,21 @@ export function executeStackFrame(
         return pop1();
       }
 
+      /**
+       * Memory operations
+       */
+      case "store": {
+        const memAddr = frame.originatingModule.memaddrs[0];
```

Comment:
Is the memory of the module always at index 0?

Reply:
yes, it is documented in the `store` instruction here: https://webassembly.github.io/spec/core/exec/instructions.html#exec-storen

and the design docs too: http://webassembly.org/docs/modules/#linear-memory-section

Reply:
Right, thanks for the clarification.